autogtp/README.md

```diff
@@ -30,10 +34,6 @@ directly.
 
 # Running
 
-Copy the compiled leelaz binary into the autogtp directory, and run
-autogtp.
-
-    cp ../build/leelaz .
-    ./autogtp
+See : [Contributing](https://github.com/wonderingabout/leela-zero/tree/minor-clearer-next#contributing)
```

Comment:
Again, this makes no real sense for the AutoGTP readme.

Reply:
again, the purpose is to regroup all commands in the same page (being the main readme page which is more noticeable and intuitive for a little knowledgeable user)

an advanced user can go to the autogtp readme to see some advanced details or options if he wants

Reply:
i added a link to autogtp readme in contributing section

Reply:
after thinking it again, i can see many cases where one person may want to compile only autogtp binary without leelaz

did an attemptive revert,
can you review again @gcp  ?